Basic-Car-Maintenance/Shared/MainView/Views/MainTabView.swift

```diff
@@ -18,28 +18,59 @@ struct MainTabView: View {
     @AppStorage("lastTabOpen") var selectedTab = TabSelection.dashboard
     @Environment(ActionService.self) var actionService
     @Environment(\.scenePhase) var scenePhase
+    @Environment(\.horizontalSizeClass) var horizontalSizeClass
+    @State var showNavigationSplitView = false
     @State private var isShowingRealTimeAlert = false
     @State var authenticationViewModel = AuthenticationViewModel()
     
     var body: some View {
-        TabView(selection: $selectedTab) {
-            DashboardView(authenticationViewModel: authenticationViewModel)
-                .tag(TabSelection.dashboard)
-                .tabItem {
-                    Label("Dashboard", systemImage: "list.dash.header.rectangle")
+        Group {
+            // If the screen is an iPhone or a split screen
+            // show a tab view, else show a NavigationSplitView
+            if horizontalSizeClass == .compact || showNavigationSplitView == false {
```

Comment:
```suggestion
            if horizontalSizeClass == .compact || !showNavigationSplitView {
```

Use `!` rather than `== false`